### ISSUE TITLE
Use sender's icon in notifications

### ISSUE THREAD

**Issue by philipzae** [IBBoard/cawbird#140]
Presently, cawbird sends notifications which have its own icon in it, and it would be better if the notification appeared with the sender's icon, so its easy to identify where it is coming from at a glance.

**Comment by IBBoard** [comment]
I've been thinking about this and I can see the point (especially for DMs), but I'm not sure that it's the correct behaviour.

I don't get notifications from other chat apps, but the screenshot in [this Gnome article](https://blogs.gnome.org/aday/2015/02/26/in-case-you-didnt-notice/) shows Polari mentioning the person and channel that sent the notification but using the Polari app. I think the reasoning is that the notification is from Polari and so you show its origin.

**Comment by philipzae** [comment]
It is the correct behavior its found everywhere. Like on phones.

![photo_2020-05-25_12-42-00](https://user-images.githubusercontent.com/4412114/82795495-28c41700-9e85-11ea-8823-44334ea41157.jpg)

And on desktops (telegram)

![Screenshot_2020-05-25_02-13-39](https://user-images.githubusercontent.com/4412114/82795588-47c2a900-9e85-11ea-9cbe-842a96aee863.png)

Regarding not know which app it will open, you can add ' (Cawbird)' at the end of the notification title.

**Comment by IBBoard** [comment]
Your first screenshot shows a Twitter icon. So it's following the "show the app icon" rule 😉 It even shows you which of your accounts it came from. Yes, it also shows the avatar, but I think the Linux notification standard only has one icon and two text strings (subject and body).

We could add "(Cawbird)" to the end. But then what if it's a long name? We could add it at the start, but then that's just junk.

The first rule of design and usability is that "it's found everywhere" does not make it correct 😉 Skeuomorphic design was found everywhere at one point. Rounded avatars are found everywhere these days. Both are really bad ideas, but they became popular and people copied them.

The correct behaviour for usability is whatever gives the user the information that they need. And the [Gnome "Human Interface Guidelines"](https://developer.gnome.org/hig/stable/notifications.html.en) say it is an "application icon", which implies it's more important (from Gnome's perspective) to tell people which app they're about to interact with.

**Comment by philipzae** [comment]
Yes it shows the twitter name, as unlike Linux, it has an additional line for the app. As you can see here, it wasn't the twitter icon but the tweet icon.
![image](https://user-images.githubusercontent.com/4412114/82808819-ccb8bd00-9e9b-11ea-94d4-79489ca5cb71.png)

I personally didn't think that ' (Cawbird)' was necessary at all, but was suggesting it if you felt it was crucial to have some indication of cawbird on the notification.

Yes I do agree that what is important is to give the user the info they need. If Gnome HIG say's its the application icon when it related specially to an IM app, then I have to say that they are wrong.

Here are some items from my notification plugin.
![image](https://user-images.githubusercontent.com/4412114/82809775-eeb33f00-9e9d-11ea-903f-bad97a81bb6f.png)
